test/e2e/smoke-test/packages/cert-manager/1.5.3/cert-manager-test.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+
+
+# Checking package is installed or not
+tanzu package installed list | grep "cert-manager.community.tanzu.vmware.com"
+
+exitcode=$?
+
+if [ "${exitcode}" == 1 ] 
+then
+version=$(tanzu package available list cert-manager.community.tanzu.vmware.com | tail -n 1 | awk '{print $2}')
+tanzu package install cert-manager --package-name cert-manager.community.tanzu.vmware.com --version "${version}"
+sleep 30s
```

Comment:
@aman556 
I don't think we need this sleep for 30 seconds. I see that `tanzu package install` handles waiting for reconcile to succeed. We could reduce this maybe. I gotta check if reconcile succeed means the pods are up and running and ready or if the pods are just created. Accordingly we can have a wait or no wait it `tanzu package install` completion happens only when everything is ready, and it's a blocking command with progress bar

Reply:
I think `tanzu package install` will take care for pods to reach ready state i noticed this thing while doing some other stuff.